### ISSUE TITLE
Add help

### ISSUE THREAD

**Issue by jjasghar** [dosman711/lita-web-title#1]
There should be a help statement:
https://github.com/dosman711/lita-web-title/blob/master/lib/lita/handlers/web_title.rb#L6

Don't make me put in a PR :metal:

**Comment by awaxa** [comment]
you lazy

#2

**Comment by jjasghar** [comment]
:+1: damn i really really am.

**Comment by awaxa** [comment]
only joking around

to be honest I will just take any excuse to use [hub](https://github.com/github/hub).  I don't know why it's so fun

**Comment by dosman711** [comment]
I don't see the value in adding a help statement for this one, though. I'll take a look at @awaxa's PR.

**Comment by jjasghar** [comment]
I disagree. Having the Help is a way for someone who doesnt have access to the Gemfile or setup to see what plugins are installed in the instance. iirc there isnt any other way to do it.

**Comment by dosman711** [comment]
Right, but what *value* does it add to any random user? If I happen to have 75 user plugins installed, is seeing this in the list really going to give me any useful information? I feel like this is one of those plugins that just kind of exists and does its thing quietly.

I am going to go ahead and merge this one, and see if @jimmycuadra has any thoughts on adding a 'list plugins' feature to Lita. I also feel like we're trying to decide some best practices and feel like he should have a say ;)

**Comment by awaxa** [comment]
I'm in favor of http based help, because receiving 75 throttled PMs on IRC is infuriating.

**Comment by dosman711** [comment]
Merged PR #2. Also bumped version and released v1.0.1 to rubygems.org.

**Comment by jimmycuadra** [comment]
My two cents: I think it's reasonable to have a help message for something like this. It responds to messages like any other handler. The convention I use variable parts of the input message to be in all caps, so I would've written the help message for this like this:

``` ruby
{ "URL" => "Responds with the title of the web page at URL" }
```
